### ISSUE TITLE
Options.py file is missing for training | Not getting the same results as indicated in the paper for ISTD+

### ISSUE THREAD

**Issue by ZeeshanMasood786** [ming053l/PhaSR#8]
**Hi Authors,** 

Thank you for sharing this amazing work. I was working on my shadow detection and shadow removal project for my Master thesis and I came across this amazing research. However I have some questions while working with thsi project,

**1.**  I tried to reproduce the results for ISTD and ISTD+ datasets. Using pretrained weight provided for ISTD+, I could only achieve these resullts **"PSNR: 31.5924, SSIM: 0.9291, RMSE: 0.9291, LPIPS: 0.0535"**  and results mentioned in the paper is **PSNR: 34.48, SSIM: 0.960.** I also tried to reproduce the results for ISTD with ISTD+ pretrained weight, but it is way too differ with the results mentioned in the paper. Results achieved are **"PSNR: 24.9569, SSIM: 0.9192, RMSE: 0.9192, LPIPS: 0.0586"** and results mentioned in the paper are **PSNR: 30.73, SSIM: 0.960.** Please provide the pretrained weight for ISTD dataset(if it is different from ISTD+)

**Modification: Padding was applied to make the test_DDP.py work, without it, the code was crashing.** 

**Let me know if anyone tried to reproduce the results and got better or equivalent results proposed by authors.** 

**2.**  While looking at the train_DDP script for the code, Options.py script is being called which contains parameters configurations for the training. Please provide that script so that someone might me able to reproduce the results with the same setup. 

**3.** Pretrained weight for SRD dataset has been provided (Thank you for sharing this). Can you please provide the PSNR and RMSE that you have achieved just in case if someone try to reproduce the results, he/she can have the baseline result to compare.  

**Thank you for sharing this amazing work and time. I am looking forward to hear from you.**

**Comment by Fengluankang99** [comment]
Hello, do you happen to have the PhaSR Model Zoo files? The Google Drive folder currently requires access approval. I have already submitted a request to the authors, but it has not been approved yet.

If sharing the files is permitted, would you mind sending me a copy or providing an alternative download link? I only need them for academic reproduction and research purposes.

Thank you very much!

<img width="1191" height="654" alt="Image" src="https://github.com/user-attachments/assets/528dcb4b-e1a1-4cb2-b61a-944526e11f8f" />
